src/Bicep.Core/Analyzers/Linter/LinterRuleBase.cs

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.Core.Analyzers.Interfaces;
+using Bicep.Core.CodeAction;
+using Bicep.Core.Configuration;
+using Bicep.Core.Diagnostics;
+using Bicep.Core.Parsing;
+using Bicep.Core.Semantics;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Bicep.Core.Analyzers.Linter
```

Comment:
Probably need to disable or hide behind debug options:

![image](https://user-images.githubusercontent.com/6913354/118567368-89013800-b72a-11eb-960a-116abf337b51.png)

Reply:
John fixed this